tiledb/sm/filter/webp_filter.cc

```diff
@@ -381,7 +381,6 @@ void WebpFilter::set_extents(const std::vector<ByteVecValue>& extents) {
     throw StatusException(Status_FilterError(
         "Tile extents too large; Max size WebP image is 16383x16383 pixels"));
   }
-  WriterTile::set_max_tile_chunk_size(extents_.first * extents_.second);
```

Comment:
set_max_tile_chunk_size should only be called from tests... Let's make the function private, then we can make a WhiteBoxWriterTile friend declaration in WriterTile and modify the tests to call set_max_tile_chunk_size from a WhiteBoxWriterTile class.

Reply:
Done :+1: